### ISSUE TITLE
Inline comments (e.g., `gcw`) in Python

### ISSUE THREAD

**Issue by IamGianluca** [numToStr/Comment.nvim#234]
Hi,

I just discovered the plugin and getting up to speed. I've seen that with some languages, the plugin is smart enough to comment out an argument in a function call:

```lua
-- from this
vim.keymap.set('n', '<space>e', vim.diagnostic.open_float, opts)
-- to this
vim.keymap.set('n',--[[  '<space>e' ]], vim.diagnostic.open_float, opts)
```

In Python, however, the plugin doesn't return a valid syntax if we comment out an argument.

```python
# from this
def my_function(a, b):
    return b
# to this
def my_function( #a, b):
    return b
```

I would have expected it to return something like this...

```python
def my_function(b):  #a):
    return b
```

I'm assuming this is because Python doesn't have inline comments, but I was wondering if there was a way to achieve something similar to what I described. It would be particularly useful when refactoring code.

**Comment by numToStr** [comment]
Yes, this is expected because python doesn't have block commentstring. And I am not sure, how is this achievable? Maybe using treesitter? But that being said, this is certainly out of scope for the plugin to support this OOTB.
